01week/datatypes.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+//1. Write a JavaScript program to display the current day and time. I was playing around with code and thought this would be the best/shortest way of geting the date infor to print.
+
+const day = (console.log(Date(Date)));
+
+
+
+//2. Write a JavaScript program to convert a number to a string.
+//Here I am trying to use the num.toString method or value.toString() method eventually i used string to change the number to a string
+const makeString=(num1)=>{
+  return String('number into a string')
+};
+console.log(makeString(87534));
+
+
+
+
+//3. Write a JavaScript program to convert a string to the number.
+//I am trying to figure this one out but can not seem to get it right. I am using string and parseInt method
+//I just simplifed it by not makeing a return function
+
+const makeNum= parseInt('10')
```

Comment:
this is a variable that stores the number 10. I need a program that converts any sting into a number